traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/ds/DeliveryService.java

```diff
@@ -96,6 +98,7 @@
 	private final boolean redirectToHttps;
 	private final DeepCachingType deepCache;
 	private String consistentHashRegex;
+	private final SortedSet<String> consistentHashQueryParams;
```

Comment:
This doesn't really need to be a sorted set since we don't have to iterate over the set in a sorted manner. Right now it's only being used for `contains()` so maybe a HashSet makes more sense.

Reply:
I thought about changing this, but the intent is to match the TO API declared representation - it's technically an array because JSON doesn't have proper sets but it _does_ sort the output.

It's probably not a big deal either way, and you have way more experience with Traffic Router (and probably just Java in general), so having said that if you want me to change it I don't have a problem doing that.

Reply:
It just seems weird to use a TreeSet when it makes more sense to use a HashSet based on how it's used in TR. TreeSets are less optimal for `contains()` compared to HashSet, and we're never iterating over the set in a sorted manner.

Reply:
Done (pending branch merge).

Reply:
only reason TO sorts them is so they don't come out differently for 2 dses with the same set.  Makes comparison more easily automated.

Reply:
It's probably best not to tie ourselves to sorting it anyway.

Reply:
I disagree.   no reason for TR to attempt to maintain the order,  but TO should.